src/test/compile-fail/E0033.rs

```diff
@@ -15,5 +15,10 @@ trait SomeTrait {
 fn main() {
     let trait_obj: &SomeTrait = SomeTrait; //~ ERROR E0425
                                            //~^ ERROR E0038
-    let &invalid = trait_obj; //~ ERROR E0033
+                                           //~| method `foo` has no receiver
+                                           //~| NOTE the trait `SomeTrait` cannot be made into an object
```

Comment:
Please move these to the left, indented *under* the erroring line, otherwise they're too wide (over 100 columns).